specification/search/data-plane/Azure.Search/stable/2020-06-30/searchindex.json

```diff
@@ -0,0 +1,1341 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "SearchIndexClient",
+    "description": "Client that can be used to query an index and upload, merge, or delete documents.",
+    "version": "2020-06-30",
+    "x-ms-code-generation-settings": {
+      "useDateTimeOffset": true,
+      "syncMethods": "None"
+    }
+  },
+  "x-ms-parameterized-host": {
```

Comment:
This looks like the Track 1 spec. Did you copy the right file?

I think the expected outcome of this PR is that the preview folder should contain specs and examples for 2020-06-30 and 2020-06-30-Preview that are both identical to the ones for 2019-05-06-Preview, other than version number.

Reply:
ah i had done 2019-Preview -> 2020-Preview and 2019-GA -> 2020-GA
Ill change to  2019-Preview -> 2020-Preview and 2019-Preview-> 2020-GA

---
In reply to: [441851314](https://github.com/Azure/azure-rest-api-specs/pull/9857#discussion_r441851314) [](ancestors = 441851314)